### PR TITLE
Enhance README and intro

### PR DESCRIPTION
**Pull request by JAORMX**
The README now contains an overall description of the lab, as well as
the agenda and prerequisites.

The index is now an introduction that's more targeted towards the
requirements for the cluster.

### CODE REVIEW

File: workshop/content/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+============
+
+This workshop provides an introduction to deploying a the
+[compliance-operator](https://github.com/openshift/compliance-operator).
+
+In this lab we'll use the E8 benchmark and evaluate through it. This benchmark
+was chosen as it's fairly lightweight compared to other benchmarks.
+
+One thing to note is that you'll need `cluster-admin` privileges in your
+cluster in order to install the operator. you'll also need an OpenShift 4.6
```
Comment (jhrozek) [inline workshop/content/index.md]: Capitalize y in "You'll"

File: workshop/content/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Introduction
+============
+
+This workshop provides an introduction to deploying a the
+[compliance-operator](https://github.com/openshift/compliance-operator).
+
+In this lab we'll use the E8 benchmark and evaluate through it. This benchmark
```
Comment (jhrozek) [inline workshop/content/index.md]: evaluate it?

**Review by jhrozek — COMMENTED** [review]
two nits, feel free to just push directly if they're fixed

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 9157595b9d4c6f1a81edc20c5faf6b3f511231d5

**Review by jhrozek — APPROVED** [review]
/lgtm

**Pull request merged**